File: c++/7kyu/square_digits.cpp
#include <string>
#include <cstdlib>

int square_digits(int num) {
  std::string n = std::to_string(num);
  std::string ans = "";
  for(auto d: n){
    int x = d - '0';
    ans += std::to_string(x * x);
  }
  return std::stoi(ans);
}

File: c#/7kyu/square_digits.cs
using System;

public class Kata
{
  public static string SquareDigits(int n)
  {
    string ans = "";
    foreach (var x in n.ToString())
    {
		  int d = int.Parse(x.ToString());
      ans += (d * d);
    }
	  return ans;
  }
}

